src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Web/Pages/_Layout.cshtml

```diff
@@ -48,6 +49,17 @@
     {
         <link rel="manifest" href="manifest.json">
     }
+    @if (isElectron)
+    {
+        <style>
+            * {
```

Comment:
plz move these lines to the below style tags.

Reply:
It seems VS bug prevents us from doing that

Reply:
I see.